libs/core/src/lib/form/form-label/form-label.component.scss

```diff
@@ -28,6 +29,15 @@ $form-label-inline-help-placement-space: 0rem;
 .fd-form-label__wrapper {
     position: relative;
     display: flex;
+    padding-bottom: $form-label-bottom-spacing;
+
+    &__align-end {
```

Comment:
I know this is Fund-ngx but we should still maintain the BEM methodology for our classes. 
`fd-form-label__wrapper__align-end` is a wrong BEM. You can't have BLOCK__ELEMENT__ELEMENT. In your case "align-end" is a modifier, so you need to have:
`fd-form-label__wrapper--align-end`